src/analogPins.cpp

```diff
@@ -0,0 +1,33 @@
+#include "analogPins.h"
+
+// Ticker to poll input readings at a fixed rate
+STM32TimerInterrupt dataUpdater(TIM2);
+
+void initAnalog() {
+    initADC(ADC1);
+
+    // start the Ticker to periodically read IO
+    if(dataUpdater.attachInterruptInterval(IO_UPDATE_PERIOD, readAnalog)) {
+        printf("starting readAnalog timer\n");
+    } else {
+        printf("problem starting readAnalog timer\n");
+    }
+}
+
+void readAnalog() {
+    lv_12V_telem = readADC(ADC_CHANNEL_6) * 3.3 *  35.1/5.1; // PA_1
+    lv_5V_telem = readADC(ADC_CHANNEL_12) * 3.3 * 15.1/5.1; // PA_7
+    lv_5V_current = readADC(ADC_CHANNEL_15) * 10; // PB_0
+    current_in_telem = readADC(ADC_CHANNEL_8) * 10; // PA_3
```

Comment:
Let's add a macro for the constant to multiply the current by, especially since it's being used by both of the current values. Something like `INA180_CURRENT_MULTIPLER`.